Utilities/DataFlow/test/test_SubframeUtils01.cxx

```diff
@@ -0,0 +1,56 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
I was under the impression that this decision was taken since `BOOST` is used via the standard macro `O2_GENERATE_TESTS`. It is successfully deployed in various places. ( I cannot find a disagreement concerning the assert statement since you agree that then one needs to take extra precautions such as a simple `#undef NDEBUG` in the translation unit of the unit test. )